content/api/commands/document.md

```diff
@@ -32,7 +32,8 @@ Pass in an options object to change the default behavior of `cy.document()`.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`cy.document()` 'yields the `window.document` object' </li></List>
+- `cy.document()` 'yields the `window.document` object.
+- `cy.document()` is a query, and it is _safe_ to chain further methods.
```

Comment:
@BlueWinds Should all Query commands be moved to live under APIs/queries to help alleviate confusion when a user is looking at the docs api face up?

Reply:
It might also provide another (or better) face up opportunity to describe the difference between query & command and how to add custom and the retry-ability differences are (at this point only through the upper docs so you might have done this...)

Reply:
Do you mean changing the URL, so that it's `/apis/query/document` rather than `/api/command/document`? I'm definitely open to the idea - we'd need to leave in place redirects so that we don't break links, but that's not hard.

I'm not sure I follow the second comment though. Can you explain what you mean again?

Reply:
Changing the slug wouldn't necessarily remove them from the same page - you'd still see them in the TOC.

![Screenshot_20221109_073631](https://user-images.githubusercontent.com/3003404/200874222-8550ee9d-2551-4e92-a5cf-027e57faf385.png)

/api/commands/..., /api/utilities/... and /api/plugins/... are already there.

@elylucas - if I wanted to move `/api/commands/as` to `/api/queries/as`, and have `/api/commands/as` redirect to the new URL, how would I go about this?

Reply:
Right, gotcha.

Reply:
Wouldn't onlinks continue to work, if we add a redirect inside the docs? Eg, `on.cypress.io/get` -> `docs.cypress.io/api/commands/get` -> `docs.cypress.io/api/queries/get`?

Reply:
> Wouldn't onlinks continue to work, if we add a redirect inside the docs? Eg, `on.cypress.io/get` -> `docs.cypress.io/api/commands/get` -> `docs.cypress.io/api/queries/get`?

They would, but good to update to keep the double redirect from happening and to be correct in the services repo.

Reply:
I've re-organized the table of contents following discussions with Brian and Emily. We now go:

Commands
- Assertions
- Actions
- Queries

and then all the other commands that aren't one of these three beneath "Commands". I've added links to relevant guides to each section.

With this hierarchy change, I'm not sure it makes sense to update the slugs any more - they're all still "Commands", some of them just also have additional modifiers ("query command", "action command").